modules/RTC/JitsiRemoteTrack.js

```diff
@@ -45,11 +48,12 @@ function JitsiRemoteTrack(
             // Nothing to do if the track is inactive.
         },
         mediaType,
-        videoType,
-        ssrc);
+        videoType);
     this.rtc = rtc;
+    this.ssrc = ssrc;
     this.ownerEndpointId = ownerEndpointId;
     this.muted = muted;
+    this.isP2P = isP2P;
```

Comment:
Conference is aware if P2P is currently active, but I don't see a problem with this field. It's for the other components (including conference) to know where does a track belong to without having to go and fetch it's parent TPC,JingleSessionPC. It doesn't do anything with this field, but it's part of it's identity.